process_pr.py

```diff
@@ -1109,20 +1149,25 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
     # to use information during the actual comment loop
     for comment in issue.get_comments():
         all_comments.append(comment)
-        if bot_cache or technical_comment:
-            continue
         if comment.user.login.encode("ascii", "ignore").decode() != cmsbuild_user:
             continue
         comment_msg = comment.body.encode("ascii", "ignore").decode() if comment.body else ""
         first_line = "".join([l.strip() for l in comment_msg.split("\n") if l.strip()][0:1])
-        if re.match(ISSUE_SEEN_MSG, first_line):
+        if (not already_seen) and re.match(ISSUE_SEEN_MSG, first_line):
             already_seen = comment
-            bot_cache = read_bot_cache(comment_msg)
-            print("Read bot cache from already seen comment:", comment)
+            bot_cache = extact_bot_cache(comment_msg)
+            if bot_cache:
+                technical_comments.append(comment)
+                continue
```

Comment:
why not simplify it to 
```
if extact_bot_cache(comment_msg):
    technical_comments.append(comment)
```

and drop the `if not bot_cache:` condition at line 1165

Reply:
and may be then `extact_bot_cache` is not needed

Reply:
`extact_bot_cache` can be removed and we just directly use the following here  ... right?
```
    seen_commits_match = REGEX_COMMITS_CACHE.search(comment_msg)
    if seen_commits_match:
        technical_comments.append(comment)
```